lib/config/schemas/gateway.config.json

```diff
@@ -0,0 +1,152 @@
+{
+  "$id": "http://express-gateway.io/models/gateway.config.json",
+  "type": "object",
+  "properties": {
+    "http": {
+      "type": "object",
+      "properties": {
+        "port": {
+          "type": "number"
+        }
+      }
+    },
+    "https": {
+      "type": "object",
+      "properties": {
+        "port": {
+          "type": "number"
+        },
+        "tls": {
+          "type": "object",
+          "additioanlProperties": {
```

Comment:
`additioanlProperties` => `additionalProperties`

Reply:
😱 
Fixed in 8617d37